config/Firebase/firebase.js

```diff
@@ -59,6 +59,19 @@ const Firebase = {
       .where('uid', '==', userId)
       .get()
   },
+  getUsers: (userId) => {
```

Comment:
This function doesn't need the userID passed to it anymore. I would also create a new function called `getUsersWithPoints` that will get all users if their points value exists and is greater than 0.